Client/Frontend/Login Management/AddCredentialViewController.swift

```diff
@@ -16,35 +16,47 @@ enum AddCredentialField: Int {
     }
 }
 
-class AddCredentialViewController: UIViewController {
+class AddCredentialViewController: UIViewController, Themeable {
+
+    var themeManager: ThemeManager
+    var themeObserver: NSObjectProtocol?
+    var notificationCenter: NotificationProtocol
 
     lazy var tableView: UITableView = .build { [weak self] tableView in
         guard let self = self else { return }
-
-        tableView.separatorColor = UIColor.theme.tableView.separator
-        tableView.backgroundColor = UIColor.theme.tableView.headerBackground
         tableView.accessibilityIdentifier = "Add Credential"
         tableView.delegate = self
         tableView.dataSource = self
         tableView.estimatedRowHeight = 44.0
         // Add empty footer view to prevent separators from being drawn past the last item.
         tableView.tableFooterView = UIView()
+        tableView.separatorInset = .zero
     }
     fileprivate weak var websiteField: UITextField!
     fileprivate weak var usernameField: UITextField!
     fileprivate weak var passwordField: UITextField!
 
     fileprivate let didSaveAction: (LoginEntry) -> Void
 
-    fileprivate lazy var cancelButton = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(cancel))
+    fileprivate lazy var cancelButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(cancel))
+        button.tintColor = themeManager.currentTheme.colors.actionPrimary
```

Comment:
This should probably only be set in applyTheme. Having it set in both places may be confusing later on if anyone wants to change it.